sdk/storage/storage-blob/samples/typescript/advanced.ts

```diff
@@ -0,0 +1,101 @@
+// Steps to run this sample
+// 1. npm install
```

Comment:
What would `npm install` do when there's no `package.json` in current directory?

Reply:
This is something that's present from the beginning.
I think it is safe to remove these comments since we have the readme specifically for samples.